posthog/temporal/delete_persons/delete_persons_workflow.py

```diff
@@ -235,8 +235,8 @@ async def run(self, inputs: DeletePersonsWorkflowInputs):
                 start_to_close_timeout=dt.timedelta(hours=2),
                 retry_policy=temporalio.common.RetryPolicy(
                     initial_interval=dt.timedelta(seconds=10),
-                    maximum_interval=dt.timedelta(seconds=60),
-                    maximum_attempts=1,
+                    maximum_interval=dt.timedelta(seconds=360),
```

Comment:
Bumped this in case we enter a retry loop, to have more time in between and not contribute to whatever is causing the retry loop.